content/Payment Platforms options - Gonzalo Mercadante.md

```diff
@@ -0,0 +1,97 @@
+The fee is 2.9% + 0.30$ per transaction, and supports subscriptions and discounts. Also,
+provides volume discounts for big companies, with more than $80.000 a month in sales.
+Credit cards: Visa, MasterCard, American Express, JCB, Discover, Diners Club, UnionPay. Accept international cards for an additional 1%. If the payment currency differs from your payout currency, a 1% conversion fee will also apply
+
+Some companies using stripe: Slack, Booking, Shoppify, Xero, National Geographic, etc.
+
+
+### Braintree
+
+Founded in 2007 and acquired by PayPal on Sep 26, 2013.
+Similar to Stripe, they have a good documentation, and let you take payments in over 130 currencies, but charges an additional 1% fee for international payments.
+The fee is 2.9% + 0.30$ per transaction and supports subscriptions and discounts. Provides volume discounts for big companies.
+Credit cards: Visa, MasterCard, American Express, JCB, Discover, Diners Club, UnionPay.
+Braintree allows you to accept Venmo and Paypal, so if you need to use Paypal in your app, then Braintree is better than Stripe. The fee for Paypal is charged directly from Paypal but it’s the same as Braintree.
+
+Some companies using Braintree: Airbnb, Uber, Github, Dropbox, Skyscanner, etc.
+
+
+### Dwolla
+
+Dwolla is an online payment solution specifically designed for better bank transactions, customer management, and account verification. It doesn't offer credit card payments support.
+Dwolla’s enterprise pricing information is available upon request. You need to contact the company sales team. Until now, they have a fixed fee per month (not depending on the transactions).
+
+Some companies using Dwolla: Goat, Sweep, Jane, Magento, Givetide
+
+
+### Paypal
+
+Paypal also allows you to handle online payment processing, and it’s a more recognized platform. On the other hand, stripe is more like a “behind the scenes” processor with a brand name customers don’t necessarily recognize, but has a long list of popular clients.
+PayPal business account fees for sales within the US it’s 2.9% + $0.30 per transaction, and for international sales, it’s 4.4% transaction fee plus a fixed fee based on currency received.
+Paypal is easier to use but Stripe is meant for businesses that need a highly customizable and tech-based solution for payment processing. If you need more features and your emphasis is specifically online payments, the winner is Stripe.
+
+
+
+
+## What are Marketplace payments?
+
+A Marketplace platform is a form of e-commerce that connects potential buyers and sellers all within one platform to help rent, buy, swap or negotiate. It allows users transferring money from their credit card or bank account to another individual’s account. There are three parties involved: an owner that allow sellers with a product or service to sell to many buyers.
```

Comment:
It was difficult (for me at least) to understand the sentence of the 3 parties involved. Maybe you can highlight the names of the 3 parties?

Reply:
I totally agree. I updated that part. Let me know if it's clearer now. Thanks!